README.md

```diff
@@ -1 +1,88 @@
-# aadhar-demo-auth
+Aadhar Demo Auth
```

Comment:
I thought for quite a while that this meant demo as in "demonstration", when I assume it means "demographic"?  I'd call it "Aadhaar Demographic Authentication".  Also, are both "Aadhaar" and "Aadhar" valid spellings?

Reply:
i think since its related with auth/authorisation one should see that as "demographic" and not "demonstration". I don't mind changing it if that makes it more clear.

Hmm.. good point. We have used Aadhar everywhere in code base but its actually Aadhaar, checked on their website to confirm. Would change it. Thanks for highlighting.